tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 84.50, "AMD": 84.45}
+COVERAGE_DICT = {"Intel": 84.35, "AMD": 84.33}
```

Comment:
Coverage dropping. Does this PR contain new paths that cannot be easily covered ?

Reply:
It does, especially in the RPC side of things and with IO operations that fail because of external reasons. Will take another look to see where can I make some improvements.